src/main/web_src/src/components/journalpostid/JournalpostidVisning.tsx

```diff
@@ -18,18 +22,21 @@ type Response = {
 
 export default ({ system, ident }: JournalpostId) => (
 	<LoadableComponent
-		onFetch={() =>
-			DollyApi.getTransaksjonid(system, ident).then(
-				({ data }): Array<Response> =>
-					data.map((id: Response) => ({
+		onFetch={() => {
+			return DollyApi.getTransaksjonid(system, ident).then(
+				({ data }): Array<Response> => {
+					return data.map((id: Response) => ({
 						transaksjonId: id.transaksjonId,
-						miljoe: id.miljoe.toUpperCase()
+						miljoe: id.miljoe
 					}))
+				}
 			)
-		}
-		render={(data: Array<Response>) =>
-			data &&
-			data.length > 0 && (
+		}}
+		renderOnError={error => {
+			return <ErrorComponent errorMessage={error} feilKomponent={'JournalpostidVisning'} />
```

Comment:
feilKomponent={'JournalpostidVisning'} skal du kunne hente ut uten å måtte sende den inn. På den måten sliper du å altid skrive samme kodelinje:
`<ErrorComponent errorMessage={error} feilKomponent={'xzy'} />`

Worst case ville jeg ha sendt den inn i loadable component.

Kontakt meg for å vite hvordan